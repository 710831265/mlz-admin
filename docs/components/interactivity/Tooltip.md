# Tooltip 文字提示

**简单的文字提示气泡框。**

### 何时使用

**鼠标移入则显示提示，移出消失，气泡浮层不承载复杂文本和操作。**

**可用来代替系统默认的 <code>title</code> 提示，提供一个 <code>按钮/文字/操作</code> 的文案解释。**

### 基本使用

<code src="./../../demo/Tooltip/normal-usage.demo.tsx" />
