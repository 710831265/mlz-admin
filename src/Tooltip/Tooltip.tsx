import React, { useState, useContext } from 'react';
import { Tooltip as AntdTooltip } from 'antd';

const Tooltip = AntdTooltip;

export default Tooltip;
