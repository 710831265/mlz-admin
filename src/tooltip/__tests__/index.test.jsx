import React, { Component } from 'react';
import { testMount, testSnapshot } from '../../../tests';
import Tooltip from '..';

describe('ðŸ§ª Tooltip', () => {
  testMount(Tooltip);
  testSnapshot(Tooltip);
});
