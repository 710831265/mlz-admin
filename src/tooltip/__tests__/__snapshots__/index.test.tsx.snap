// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`🧪 Tooltip snapshot test 快照匹配 1`] = `"<span></span>"`;
